crates/gen/Cargo.toml

```diff
@@ -1,17 +1,16 @@
 [package]
-name = "winrt_gen"
+name = "winrt-gen"
```

Comment:
I believe this means we'll have to create a new crate on Crates.io. Is that what we want?

Reply:
Any reason to switch to dashes? I settled on underscores for the package names, to match what is required in the `use` statements.

Reply:
We had a mix of dashes and underscores - I just settled on dashes for consistency and because I wasn't sure whether the name of the `cargo-winrt` crate mattered. The others don't.